x-pack/elastic-agent/CHANGELOG.next.asciidoc

```diff
@@ -124,3 +124,4 @@
 - Enable configuring monitoring namespace {issue}26439[26439]
 - Communicate with Fleet Server over HTTP2. {pull}26474[26474]
 - Support Node and Service autodiscovery in kubernetes dynamic provider. {pull}26801[26801]
+- Add new --enroll-delay option for install and enroll commands. {pull}27118[27118]
```

Comment:
@andresrc As we publish this changelog, should we become more verbose here to have more details on what this is about?